pkg/reconciler/security/istio/eventpolicybinding/resources/httppolicy.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	cehttp "github.com/cloudevents/sdk-go/v2/protocol/http"
+	"github.com/google/knative-gcp/pkg/apis/security/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/pkg/kmeta"
+)
+
+// MakeHTTPPolicy generates a HTTPPolicy based on the given EventPolicy.
+func MakeHTTPPolicy(b *v1alpha1.EventPolicyBinding, p *v1alpha1.EventPolicy) *v1alpha1.HTTPPolicy {
+	hp := &v1alpha1.HTTPPolicy{
+		ObjectMeta: metav1.ObjectMeta{
+			// TODO: Consider use random suffix.
```

Comment:
yeah, maybe you can just do "http-%s-%s" and have p.Name and p.UUID or something...
And use kmeta.ChildName to make sure is a valid name

Reply:
I will address this separately